msi/build/build.ps1

```diff
@@ -93,7 +93,14 @@ Get-ChildItem .\bin\Release -Filter *.msi -Recurse |
             Write-Host "Signing installer"
             # always disable tracing here
             Set-PSDebug -Trace 0
-            signtool sign /v /f $env:PKCS12_FILE /p $env:SIGN_STOREPASS /t http://timestamp.verisign.com/scripts/timestamp.dll /d "Jenkins Automation Server ${JenkinsVersion}" /du "https://jenkins.io" $_.FullName
+            while($True) {
+                $p = Start-Process -Wait -PassThru -NoNewWindow -FilePath "signtool.exe" -ArgumentList "sign /v /f `"${env:PKCS12_FILE}`" /p ${env:SIGN_STOREPASS} /t http://timestamp.verisign.com/scripts/timestamp.dll /d `"Jenkins Automation Server ${JenkinsVersion}`" /du `"https://jenkins.io`" $_.FullName"
+                $p.WaitForExit()
+                # we will retry until we get a good exit code
+                if($p.ExitCode -eq 0) {
```

Comment:
Do we not want to give up at some point or is there a timeout higher up that will handle this?

Should we sleep in between attempts to give the server more time to recover?

Reply:
A timeout would probably be good, it would just result in a failure, so not a bid deal. My guess is its not the server that is having issues, but the client accessing the server. Those timestamp servers have insane uptimes and response rates.